### PR TITLE
[ENG-1170] Adding draft_registration, new schema blocks

### PR DESCRIPTION
**Pull request by corbinSanders**
Schema blocks includes some changes to the API. These include GET methods to schema blocks, GET and PATCH to draft_registrations, and some modification of some fields on the registration end points. These file changes reflect those API changes.

https://openscience.atlassian.net/browse/ENG-1170

### CODE REVIEW

File: swagger-spec/schemas/definition.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+type: object
```
Comment (aaxelb) [inline swagger-spec/schemas/definition.yaml]: seems like `/schemas/definition.yaml` should be the definition of the schema type, and the schema-block definition should be in `/schemas/schema_block_definition.yaml` (or something like that -- just following what seems to be a convention)

File: swagger-spec/swagger.yaml [hunk 2]
```diff
@@ -1306,6 +1315,16 @@ paths:
   /registrations/{registration_id}/wikis/:
     $ref: 'registrations/wikis_list.yaml'
 
+  #####################
+  #   SCHEMAS   #
+  #####################
+
+  /schemas/registrations/{schema_id}/schema_blocks/:
```
[the remark this answers is not in the document]
Comment (brianjgeiger) [reply swagger-spec/swagger.yaml]: If the `/schemas/` endpoints are necessary for properly using schema blocks, then yeah, we should do this here.

[the remark this answers is not in the document]
Comment (aaxelb) [reply swagger-spec/swagger.yaml]: yeah, starting from a registration (or draft), the path is now:
- follow `schema` relation to `/schemas/registrations/{id}`
- follow the schema's `schema_blocks` relation to `/schema/registrations/{id}/schema_blocks/`
- use schema blocks to render/edit/validate/whatever the original registration's (or draft's) `registration_responses`

[the remark this answers is not in the document]
Comment (corbinSanders) [reply swagger-spec/swagger.yaml]: I marked all metaschema endpoints as deprecated, and added the other schema endpoints

File: swagger-spec/nodes/registrations_list.yaml [hunk 3]
```diff
@@ -218,14 +218,13 @@ post:
     &nbsp;&nbsp;&nbsp;&nbsp;`draft_registration`
 
 
-    &nbsp;&nbsp;&nbsp;&nbsp;`registration_choice`
 
+    #### Optional
 
-    &nbsp;&nbsp;&nbsp;&nbsp;`lift_embargo` (Only required when `registration_choice` is "embargo")
+    &nbsp;&nbsp;&nbsp;&nbsp;`children` (If left unspecified all children will be registered).
```
Comment (aaxelb) [inline swagger-spec/nodes/registrations_list.yaml]:
```suggestion
    &nbsp;&nbsp;&nbsp;&nbsp;`included_node_ids` (If left unspecified all child nodes will be registered).
```

**Comment by aaxelb** [comment]
oh, also, got a conflict 😖

**Pull request merged**